content/en/blog/2021/blog-mermaid.md

```diff
@@ -0,0 +1,172 @@
+linkTitle: "Improve your documentation with diagrams using Mermaid.js"
+date: 2021-11-01
+---
+By Chris Metz and Tim Bannister
+
+---
+
+Topics covered in this blog.
+
+![Why are diagrams useful for documentation? Use mermaid.js / examples](/img/mermaid-blog-outline.svg)
+
+<!--- added padding to align text with diagram above --->
+<p style="padding-left: 200px; font-weight: bold"><a href="https://mermaid-js.github.io/mermaid-live-editor/edit/#eyJjb2RlIjoiJSV7aW5pdDp7XCJ0aGVtZVwiOlwibmV1dHJhbFwifX0lJVxuZmxvd2NoYXJ0IExSXG4gICAgQltXaHkgYXJlIGRpYWdyYW1zPGJyPnVzZWZ1bCBmb3IgZG9jdW1lbnRhdGlvbj9dIC0tPiBDW1VzZSBNZXJtYWlkLmpzXVxuICAgIEMgLS0-IERbRXhhbXBsZXNdXG5cbiAgICAiLCJtZXJtYWlkIjoie1xuICBcInRoZW1lXCI6IFwiZGVmYXVsdFwiXG59IiwidXBkYXRlRWRpdG9yIjpmYWxzZSwiYXV0b1N5bmMiOnRydWUsInVwZGF0ZURpYWdyYW0iOmZhbHNlfQ">Live editor link: Topics Covered</a></p>
+
+---
+
+## Why are diagrams useful for documentation?
+
+* **Friendly landing spot**. Greeting readers with a page full of text can be intimidating to those new to Kubernetes, software engineering and tech writing.
+
+* **Faster grasp of concepts.** A diagram can serve as a visual roadmap for details covered in the accompanying text.
+
+* **Better retention.** Humans remember pictures better than words.
+
+Most importantly, readers **acquire the confidence to explore further**. "Wow! Okay I get this. What's next? Maybe I can contribute."
+
+There are many drawing tools to choose from. Traditionally, you create a diagram, save it as png or svg file and then embed the file in the documentation. This takes time, skills vary, and complicates collaboration between contributors and reviewers.
+
+What if we told you there is a package that allows you to generate simple diagrams using a markdown-like syntax?  
+
+---
+
+## Mermaid.js
+
+[Mermaid](https://mermaid-js.github.io/mermaid/#/) is a package for generating diagrams using markdown-ish text in markdown files. Here are some of the features we love about Mermaid:
+
+* **Inline code syntax** you can copy into the page markdown. Indeed, you should treat Mermaid code as documentation text subject to the usual PR processes. 
+
+* **[On-line live editor](https://mermaid-js.github.io/mermaid-live-editor)** to dynamically create and edit diagrams. The live editor displays the code and the rendered diagram. __TIP: You should always use the live editor to draw diagrams.__
+
+* **Share diagram URLs**. The live editor generates a URL for each saved diagram. You can share this link with colleagues to collaborate on diagram creation and editing.
+
+* **Generate svg image files**. If your site doesn't support Mermaid, but can handle svg image files as most do, then just ask the live editor to generate an svg file. __HINT: we use svg files in this blog because the contributor-site does not support Mermaid.__
+
+You can choose from different formats including flowchart, sequence and state. Mermaid works out of the box with the [K8s.io](https://kubernetes.io/) website. 
```

Comment:
```suggestion
You can choose from different formats including flowchart, sequence and state.
The main [Kubernetes website](https://kubernetes.io/) is set up so that Mermaid works out of the box.
```